includes/Settings/SettingsUpgrade.php

```diff
@@ -304,12 +305,16 @@ public function get_extension_license_infos( $ignore_cache = false ) {
 			if ( ! empty( $bundle_upgrade_link ) && $license_status == 'valid' ) {
 				$license_info[$extension]['url'] = $bundle_upgrade_link;
 			} else {
+				$license = ! empty( $license_info[$extension]['license_key'] ) ? $license_info[$extension]['license_key'] : ( ! empty( $license_info['pro']['license_key'] ) ? $license_info['pro']['license_key'] : $license_info['templates']['license_key'] );
```

Comment:
Maybe you don't need this line?

Reply:
If the license is empty, wouldn't it be better to provide the product link instead?

Reply:
@alexmigf It makes sense, but do you think we should redirect the user to the PRO plugin product link or to the bundle?
The PRO is the next step when it comes to upgrading from free, but bundle also makes sense because it is a better deal.

Reply:
I believe the Professional should link to its respective product, and the Bundle should link to itself as well. Linking both to the Bundle might be problematic—it could give customers the impression that we're trying to mislead them.